scripts/ConfigureEvaluationTests.ps1

```diff
@@ -0,0 +1,45 @@
+param (
```

Comment:
It'd be great to have explanations for this file and the input args similar to other ps1s in this folder.

Also, if this file is M.E.AI.Eval test specific, then would it make more sense to locate it in the project's folder?

Reply:
FYI @peterwald - perhaps we should move this down to the test folder per @RussKie's recommendation?

Reply:
Actually, since this is used by a couple of test projects, we will need to split out two copies if we want to move it down... Keeping it in the same location (and adding explanations) may work better from that perspective.

Reply:
I think we can leave it in this folder for now - we can figure out alternate location in the future if needed. (We will likely be making more changes to this and other scripts as part of enabling the above tests to run in CI.)

Reply:
Sounds good

> It does indeed apply across several M.E.AI.Eval projects, so if we move it down, we'd need to find a good intermediate folder under Tests or something.

...or it could be linked ;)
E.g., `<Compile Include="$(MSBuildThisFileDirectory)\..\..\<path>\...ps1" LinkBase="scripts" />`